pkg/commands/app.go

```diff
@@ -166,6 +166,24 @@ var (
 		EnvVars: []string{"TRIVY_CACHE_BACKEND"},
 	}
 
+	redisBackendCACert = cli.StringFlag{
+		Name:    "redis-ca",
+		Usage:   "redis ca file location, if using redis as cache backend",
+		EnvVars: []string{"TRIVY_REDIS_BACKEND_CA"},
+	}
+
+	redisBackendCert = cli.StringFlag{
+		Name:    "redis-cert",
+		Usage:   "redis certificate file location, if using redis as cache backend",
+		EnvVars: []string{"TRIVY_REDIS_BACKEND_CERT"},
+	}
+
+	redisBackendKey = cli.StringFlag{
+		Name:    "redis-key",
+		Usage:   "redis key file location, if using redis as cache backend",
+		EnvVars: []string{"TRIVY_REDIS_BACKEND_KEY"},
+	}
+
```

Comment:
Can we start with hidden flags? I'm suggesting `Hidden = true`. I didn't see many requests in https://github.com/aquasecurity/trivy/issues/1166. We want to keep Trivy's options as few as possible for simple usage. A lot of flags overwhelms users. If many users will complain about it, we can switch `Hidden = false` then.

Reply:
> We want to keep Trivy's options as few as possible for simple usage. A lot of flags overwhelms users

Sure, i agree with that.